sdk/src/types/block/output/unlock_condition/expiration.rs

```diff
@@ -76,16 +77,15 @@ pub(crate) mod dto {
         #[serde(rename = "type")]
         pub kind: u8,
         pub return_address: AddressDto,
-        #[serde(rename = "unixTime")]
-        pub timestamp: u32,
+        pub slot_index: u64,
```

Comment:
Can we use `SlotIndex` here? And if so, does it properly serialize as a string?

Reply:
Accidentally resolved, but this should still be done

Reply:
Done